app/bundles/CoreBundle/Helper/Chart/SeriesPieChart.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Mautic\CoreBundle\Helper\Chart;
```

Comment:
Strict typing is missing.

```suggestion

declare(strict_types=1)

namespace Mautic\CoreBundle\Helper\Chart;
```

Reply:
Ok, I added strict_types.